### PR TITLE
feat: Update the about page to include a brief history of the project.

### PR DESCRIPTION
**Pull request by Laphatize**
### Description

I included a brief history of the project in the /about section of the website.

Screenshot showcasing this:
Screenshot of Page

### Issues Resolved

Addresses a point mentioned in https://github.com/valkey-io/valkey-website/issues/13.
>  Content: About Page (/about/) MVP: 1 paragraph that describes the history of the project

### Check List
- [X] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -2,4 +2,10 @@
...
 title: About
----
\ No newline at end of file
+---
+
+
+
+<h1>History of Valkey</h1>
+
+Valkey is an open source fork of Redis, created in March 2023 after Redis announced a change to a more <a href="https://redis.io/blog/redis-adopts-dual-source-available-licensing/">restrictive license</a>. The project was initiated by <a href="https://github.com/madolson">Madelyn Olson</a>, a former Redis core maintainer from AWS, along with other Redis contributors who disagreed with the license change. Valkey quickly gained support from major tech companies such as AWS, Google, Oracle, Ericsson, and Snap, with the Linux Foundation providing a new home for the fork to maintain the original open source license. Despite the emergence of other Redis alternatives and forks, Valkey has gained significant traction due to its strong backing from cloud companies that rely heavily on Redis for their services, making it the most promising option for the near future.
```
Comment (stockholmux) [inline about/index.html:11]: - There are a lot of opinions about the word 'fork'. Maybe: `Valkey is a open source continuation of Redis created`
- `more restrictive license` change to `non-open source license`.

**Review by stockholmux — CHANGES_REQUESTED** [review]
This is fantastic work @Laphatize!

I'll accept it with the changes suggested, but if you have time convert it from an HTML file to markdown and make each sentence one line.

**Comment by Laphatize** [comment]
Should be all taken care of. Sorry it took me so long - was swamped with homework.

**Review by stockholmux — APPROVED** [review]
LGTM

**Comment by stockholmux** [comment]
Congrats @Laphatize! Glad to have you as a contributor.

**Pull request merged**